app/components/course-page/course-stage-step/second-stage-instructions-card/implement-solution-step.hbs

```diff
@@ -34,23 +23,30 @@
           {{/if}}
         {{/each}}
       </div>
-
-      {{#if @shouldRecommendLanguageGuide}}
-        <p class="prose dark:prose-invert prose-compact">
-          For a more detailed explanation on how this solution works, view the
-          <a href="#language-guide-card">{{@repository.language.name}} Guide</a>
-          card.
-        </p>
-      {{/if}}
     </:content>
 
     <:overlay>
-      <PrimaryButton {{on "click" this.handleRevealSolutionButtonClick}} class="self-center" data-test-reveal-solution-button>
+      <SecondaryButton
+        class="self-center bg-white/80 dark:bg-black/50 backdrop-blur-3xl group-hover:text-teal-600 group-hover:border-teal-600"
```

Comment:
@andy1li should the `group-hover` classes be the same for dark mode and light mode? This looks a bit weird for dark mode, as if the button becomes more "inactive"

Reply:
https://github.com/user-attachments/assets/5182d9db-97ef-4f55-8d20-9fbfb90ad313

Reply:
Tweaked design a bit, all good here